usaspending_api/config/tests/unit/test_config.py

```diff
@@ -493,6 +337,389 @@ def factory_func():
     },
 ]
 
+def test_config_values():
```

Comment:
Is this actually even testing anything?

Reply:
It's really there more for a quick eyeball regression test. But it does confirm that an object named `CONFIG` exists and has values. Well, I guess I could put some not empty asserts on the dict.

Reply:
Added asserts.